vllm/distributed/parallel_state.py

```diff
@@ -342,6 +346,62 @@ def broadcast_object_list(self,
                                                 group=self.device_group)
         return obj_list
 
+    def send_object(self,
+                    obj: Any,
+                    dst: int,
+                    group: Optional[ProcessGroup] = None) -> None:
```

Comment:
we know we are sending python object here, so it is reasonable to just use `cpu_group`.

Reply:
Done